contributors/devel/owners.md

```diff
@@ -1,100 +1,232 @@
+    applies an `lgtm` label to the PR
+- Phase 2: Humans approve the PR
+  - The PR **author** `/assign`'s all suggested **approvers** to the PR, and optionally notifies
+    them (eg: "pinging @foo for approval")
+  - The PR **author** can be their own **approver** (assuming they are listed in the appropriate
+    OWNERS files)
+  - **Approvers** look for holistic acceptance criteria, including dependencies with other features,
+    forwards/backwards compatibility, API and flag definitions, etc
+  - If the code changes look good to them, an **approver** types `/approve` in a PR comment or
+    review; if they change their mind, they `/approve cancel`
+  - `mungegithub` ([@k8s-merge-robot](https://github.com/k8s-merge-robot/)) updates its comment in
+    the PR to indicate which **approvers** still need to approve
+  - Once all **approvers** (one from each of the previously identified OWNERS files) have approved,
+    `mungegithub` ([@k8s-merge-robot](https://github.com/k8s-merge-robot/)) applies an `approved`
+label
+- Phase 3: Automation merges the PR
+  - All required labels are present (eg: `lgtm`, `approved`)
+  - All required status checks for the PR are verified as green
+  - The PR is merged
+
+## Quirks of the Process
+
+There are a number of behaviors we've observed that while _possible_ are discouraged, as they go
+against the intent of this review process.  Some of these could be prevented in the future, but this
+is the state of today.
+
+- An **approver**'s `/lgtm` is simultaneously interpreted as an `/approve`.
+  - This goes against the idea of having two sets of eyes on a PR, and can be confusing to an
+    observer of the PR.
+  - Instead, explicitly write out `/lgtm` and `/approve` to help observers, or save the `/lgtm` for
+    a **reviewer**
+  - This is a sign that there are too many **approvers** who are also **reviewers**; growing the
+    `reviewers` list would help with this
+- An **approver** can `/approve no-issue` to bypass the requirement that PR's must have linked
+  issues
+  - There is disagreement within the community over whether requiring every PR to have a linked
+    issue provides value
+  - Protest is being expressed in the form of overuse of `/approve no-issue`
+  - Instead, suggest to the PR **author** that they edit the PR description to include a linked
+    issue
+  - This is a sign that we need to actually deliver value with linked issues, or be able to define
+    what a "trivial" PR is in a machine-enforceable way to be able to automatically waive the linked
+issue requirement
+- Technically, anyone who is a member of the kubernetes GitHub organization can drive-by review a
+  PR.
+  - How else can you demonstrate your candidacy to be a reviewer other than by reviewing?  This seems 
+    like something we should continue to allow.  However...
+  - This goes against the idea of specifying **reviewers** in the first place, to ensure that the
```

Comment:
That's not the current state of the world today. If someone comments `/lgtm` on a PR, prow will attempt to assign the PR to them. If they are a collaborator or org member that is able to have the PR assigned to them, then it will do that, and then apply the LGTM label.

If you're suggesting that this be changed @thockin, then we should get an issue in test-infra to do so.